### PR TITLE
Updated resources.json with my resource

### PR DESCRIPTION
**Pull request by evgenyfedorenko**

### CODE REVIEW

**Comment by timdeschryver** [comment]
Glad to see you here @evgenyfedorenko 🙂
I think there's something wrong because the build is saying it isn't valid JSON, could you take a look please?

**Comment by evgenyfedorenko** [comment]
can you please take a look what is the conflict?

File: src/assets/resources.json [hunk 1]
```diff
@@ -529,6 +529,20 @@
...
       }
     },
+    {
+      "title": "Cheatsheet for developing lightning fast progressive Angular applications.",
+      "description": "This document contains a list of practices that will help us boost the performance of our Angular applications. 'Angular Performance Checklist' covers different topics - from server-side pre-rendering and bundling of our applications to runtime performance and optimization of the change detection performed by the framework.",
+      "author": {
+        "name": "Minko Gechev",
+        "profile": "https://github.com/mgechev"
+      },
+      "category": "library",
```
Comment (timdeschryver) [inline src/assets/resources.json]:
```suggestion
      "category": "article",
```

**Comment by timdeschryver** [comment]
Thanks for your contribution @evgenyfedorenko  🎉
Once the master build is ready it will become available at https://timdeschryver.github.io/hacktoberfest-2019-angular/

**Pull request merged**